src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -434,6 +437,14 @@ <h3 class="course-detail__label">{% trans 'License for the content created by co
     </div>
     {% endblock fragment_relations %}
 
+    {% else %}
+    <div class="banner banner--error banner--rounded" role="alert">
+        <svg class="banner__icon"><use href="#icon-cross" /></svg>
+        <p class="banner__message">
+            {% trans "A course object is missing on this course page. Please select another page template." %}
```

Comment:
+If what you need is a course page, you need to create it via <a href="..." >the wizard</a> and choose "create course page"